skraper.py

```diff
@@ -1,12 +1,14 @@
-from typing import Iterable, Union
```

Comment:
This file is getting big. Maybe split into a folder ?

Reply:
the file has never been commited in the first place as it was for local devs only. We are now adding this notion to the repo.
Update read me on how to use it in local?

Reply:
moved to a folder